tests/MongoDB.Driver.Tests/Specifications/atlas-data-lake/prose-tests/AtlasDataLakeProseTests.cs

```diff
@@ -0,0 +1,115 @@
+            var connectionString = CoreTestConfiguration.ConnectionString;
+            var username = connectionString.Username;
+            var password = connectionString.Password;
+            var source = connectionString.AuthSource;
+
+            var settings = DriverTestConfiguration.Client.Settings.Clone();
+            settings.Credential = MongoCredential.FromComponents(mechanism: "SCRAM-SHA-1", source, username, password);
+
+            var client = DriverTestConfiguration.CreateDisposableClient(settings);
+            client.GetDatabase("admin").RunCommand<BsonDocument>(new BsonDocument("ping", 1));
+        }
+
+        [SkippableFact]
+        public void Driver_should_connect_to_AtlasDataLake_with_SCRAM_SHA_256()
+        {
+            RequireEnvironment.Check().EnvironmentVariable("ATLAS_DATA_LAKE_TESTS_ENABLED");
+            RequireServer.Check().Supports(Feature.ScramSha256Authentication);
+
+            var connectionString = CoreTestConfiguration.ConnectionString;
+            var username = connectionString.Username;
+            var password = connectionString.Password;
+            var source = connectionString.AuthSource;
+
+            var settings = DriverTestConfiguration.Client.Settings.Clone();
+            settings.Credential = MongoCredential.FromComponents(mechanism: "SCRAM-SHA-256", source, username, password);
+
+            var client = DriverTestConfiguration.CreateDisposableClient(settings);
+            client.GetDatabase("admin").RunCommand<BsonDocument>(new BsonDocument("ping", 1));
+        }
+
+        [SkippableFact]
+        public void KillCursors_should_return_expected_result()
+        {
+            RequireEnvironment.Check().EnvironmentVariable("ATLAS_DATA_LAKE_TESTS_ENABLED");
+            RequireServer.Check().Supports(Feature.KillCursorsCommand);
+
+            var databaseName = "test";
+            var collectionName = "driverdata";
+
+            var eventCapturer = new EventCapturer()
+                .Capture<CommandStartedEvent>(x => "killCursors" == x.CommandName)
+                .Capture<CommandSucceededEvent>(x => new[] { "killCursors", "find" }.Contains(x.CommandName));
+
+            using (var client = DriverTestConfiguration.CreateDisposableClient(eventCapturer))
+            {
+                var cursor = client
+                    .GetDatabase(databaseName)
+                    .GetCollection<BsonDocument>(collectionName)
+                    .Find(new BsonDocument(), new FindOptions { BatchSize = 2 })
+                    .ToCursor();
+
+                var findCommandSucceededEvent = eventCapturer.Events.OfType<CommandSucceededEvent>().First(x => x.CommandName == "find");
+                var findCommandResult = findCommandSucceededEvent.Reply;
+                var cursorId = findCommandResult["cursor"]["id"].AsInt64;
+                var cursorNamespace = CollectionNamespace.FromFullName(findCommandResult["cursor"]["ns"].AsString);
+
+                cursor.Dispose();
```

Comment:
I am compelled to leave it as is, because this code clearly represents the flow of the prose test ([link](https://github.com/mongodb/specifications/blob/master/source/atlas-data-lake-testing/tests/README.rst#prose-tests)):
> Observe the CommandSucceededEvent event for the find command and extract the cursor's ID and namespace from the response document's cursor.id and cursor.ns fields, respectively.
**Destroy the cursor object** and observe a CommandStartedEvent and CommandSucceededEvent for the killCursors command.

The `using` statement makes code less verbose in this case.